src/hooks/useArNSContract/useArNSContract.tsx

```diff
@@ -0,0 +1,49 @@
+import { LocalFileSystemDataProvider } from '../../services/arweave/LocalFilesystemDataProvider';
+import { useStateValue } from '../../state/state';
+import { useState, useEffect } from 'react';
+
+const localContractAddress = 'bLAgYxAdX2Ry-nt6aH2ixgvJXbpsEYm28NgJgyqfs-U';
```

Comment:
when hard-coding values, it's common practice to set them as all caps

```suggestion
const ARNS_SOURCE_CONTRACT_ID = 'bLAgYxAdX2Ry-nt6aH2ixgvJXbpsEYm28NgJgyqfs-U';
```